src/spec/sync/utils/product.spec.coffee

```diff
@@ -1390,6 +1390,26 @@ describe 'ProductUtils', ->
         expect(actual).toEqual(expected)
 
       it 'should build a transitionState action for a state change', ->
+        # test with { state: null }
+        newRef = _.extend({}, @NEW_REFERENCE, { state: null })
```

Comment:
There actually is a test for this case [a few lines further](https://github.com/sphereio/sphere-node-sdk/pull/138/files#diff-2a9c8ef1ff08242733bd0f96f869ada8R1403) 😉